service.go

```diff
@@ -228,13 +243,50 @@ func newServiceManager(c *Server, o *Overlay) *serviceManager {
 	return s
 }
 
+// openDb opens a database at `path`. It creates the database if it does not exist.
+func openDb(path string) (*bolt.DB, error) {
+	db, err := bolt.Open(path, 0600, nil)
```

Comment:
what happens if the path doesn't exist? Will it be created?

Reply:
If the database does not exist, it will be created. If the parent path does not exist, e.g. `.local/share/conode`, then an error should be returned. Which I believe is fine because this function should only be called after `contextDataPath` is initialised. I can add a comment in there to clarify.

Reply:
OK, small comment is enough. I was not sure anymore if we correctly create the `.local/sahre/conode`-path...